### PR TITLE
Shape: Private Conversations Chat Menu

### PR DESCRIPTION
**Pull request by fcolarich**
# Pull Request Description

### Prerequisites
- [ ] List any required setup steps
- [ ] Include environment/configuration requirements

### Test Steps
1. First step
2. Second step
3. Expected result after step 2
4. ...

### Additional Testing Notes
- Note any edge cases to verify
- Mention specific areas that need careful testing
- List known limitations or potential issues

## Quality Checklist
- [ ] Changes have been tested locally
- [ ] Documentation has been updated (if required)
- [ ] Performance impact has been considered
- [ ] For SDK features: Test scene is included

## Code Review Reference
Please review our [Code Review Standards](https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md) before submitting.

### CODE REVIEW

File: Explorer/Assets/DCL/Chat/ChatView.cs [hunk 1]
```diff
@@ -106,79 +78,81 @@ public class ChatView : ViewBase, IView, IViewWithGlobalDependencies, IPointerEn
...
 
         [Header("Audio")]
-        [SerializeField]private AudioClipConfig chatReceiveMessageAudio;
+        [SerializeField] private AudioClipConfig chatReceiveMessageAudio;
         [SerializeField] private AudioClipConfig chatReceiveMentionMessageAudio;
 
-        /// <summary>
-        /// Raised when the mouse pointer hovers any part of the chat window.
+                /// <summary>
+        ///     Raised when the mouse pointer hovers any part of the chat window.
```
Comment (QThund) [inline Explorer/Assets/DCL/Chat/ChatView.cs:85]: Why does Rider add that tab in summaries? Is it a rule in the automated code styling? Can we remove it?

File: Explorer/Assets/DCL/Chat/History/ChatChannel.cs [hunk 2]
```diff
@@ -35,18 +35,15 @@ public enum ChatChannelType
         public struct ChannelId
         {
             public string Id { get; }
+            public ChatChannelType Type { get; }
```
Comment (QThund) [inline Explorer/Assets/DCL/Chat/History/ChatChannel.cs:38]: I thought about this while working on my branch and in my opinion we should store the type in the ChatChannel and leave the ChannelId as small as possible, just a string. In my branch I've already done that.

File: Explorer/Assets/DCL/Chat/ChatView.cs [hunk 3]
```diff
@@ -424,16 +438,13 @@ public void InsertTextInInputBox(string text)
...
         /// </summary>
         public void RefreshMessages()
         {
             int pendingMessages = currentChannel!.Messages.Count - currentChannel.ReadMessages;
 
-            if (pendingMessages > 0)
-            {
-                chatMessageViewer.ShowSeparator(pendingMessages + 1);
-            }
+            if (pendingMessages > 0) { chatMessageViewer.ShowSeparator(pendingMessages + 1); }
```
Comment (QThund) [inline Explorer/Assets/DCL/Chat/ChatView.cs:447]: I hate 1-line ifs because I think they worsen readability, but you can leave it as it is xD

File: Explorer/Assets/DCL/Chat/ChatView.cs [hunk 4]
```diff
@@ -598,7 +576,7 @@ private void OnChatMessageOptionsButtonClicked(string messageText, ChatEntryView
             closePopupTask.TrySetResult();
             closePopupTask = new UniTaskCompletionSource();
 
-            ChatEntryMenuPopupData data = new ChatEntryMenuPopupData(
+            var data = new ChatEntryMenuPopupData(
```
Comment (QThund) [inline Explorer/Assets/DCL/Chat/ChatView.cs:579]: ¿? I guess it was the automatic code styling?

File: Explorer/Assets/Scenes/Main.unity
Comment (QThund) [inline Explorer/Assets/Scenes/Main.unity:1]: Added by mistake I guess.

**Review by QThund — APPROVED** [review]
Great refactor! Although it is correct, there are 2 things to pay attention to: the main scene and the ChannelId modification.

File: Explorer/Assets/DCL/Chat/ChatView.cs (hunk 4 above)
Comment (fcolarich) [reply Explorer/Assets/DCL/Chat/ChatView.cs:579]: yeah :S

File: Explorer/Assets/DCL/Chat/ChatView.cs (hunk 3 above)
Comment (fcolarich) [reply Explorer/Assets/DCL/Chat/ChatView.cs:447]: its the automatic code thing... I fixed a couple of things it did (like move all events to the bottom or shit like that) but others I didnt notice...

File: Explorer/Assets/DCL/Chat/History/ChatChannel.cs (hunk 2 above)
Comment (fcolarich) [reply Explorer/Assets/DCL/Chat/History/ChatChannel.cs:38]: What I didnt like was having to parse the channel string each time I needed something from it :S

File: Explorer/Assets/DCL/Chat/ChatView.cs (hunk 1 above)
Comment (fcolarich) [reply Explorer/Assets/DCL/Chat/ChatView.cs:85]: yeah its a rule in our automated Code Style Reformat DCL, it needs various adjustments tbh...
